django_project/frontend/src/pages/Admin/ContextLayer/StyleConfig/RasterCogLayer/index.jsx

```diff
@@ -209,10 +209,22 @@ export default function RasterCogLayer(
           }
         }
         options={
+          // dynamicClassificationChoices.filter((idx, choice) => idx < 3).map(choice => ({
```

Comment:
You can filter the label with the necessary one
dynamicClassificationChoices.filter(classification => ["Natural breaks"].includes(classification.label))